### PR TITLE
Add display_html property to demos

### PR DESCRIPTION
**Pull request by onishiweb**

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -19,12 +19,14 @@
 		{
 			"name": "palette",
 			"template": "demos/src/palette.mustache",
-			"description": ""
+			"description": "",
+			"display_html": "false"
```
Comment (rowanmanning) [inline origami.json]: Any reason why these are `"false"` as a string rather than a boolean?

Comment (onishiweb) [inline origami.json]: erm, only because I copied the line above and forgot that boolean's don't need `""`'s... And it's Monday. 

Will fix now.

**Comment by rowanmanning** [comment]
Looks good 👍

File: origami.json (hunk 1 above)
Comment (rowanmanning) [inline origami.json]: Mondays are killers 😴

**Pull request merged**